src/fshtypes/FshQuantity.ts

```diff
@@ -10,7 +10,10 @@ export class FshQuantity extends FshEntity {
   toString(): string {
     let str = this.value.toString();
     if (this.unit?.code != null) {
-      str += ` '${this.unit.code}'`;
+      str +=
+        this.unit.system != null && this.unit.system != 'http://unitsofmeasure.org'
+          ? ` ${this.unit.system}#${this.unit.code}`
+          : ` '${this.unit.code}'`;
```

Comment:
We now allow someone to do something like this:
```
Extension: Example
* valueQuantity = 12 #foo "hello"
```
To set the `value`, `code`, and `display`, but in this case I think the `toString` function would give something like this:
```
12 'foo' "hello"
```
Which seems off since that is the syntax for assuming UCUM as the system, but in this case there is no system. I think we would prefer it to be `12 #foo "hello"`. 

Or we disallow what I am doing altogether and require a `system` when setting a Quantity in this way. I think @cmoesel would know better than me if that is what we want to do.

Reply:
The spec says that if you have a `code`, you must have a `system` too:
<img width="810" alt="image" src="https://user-images.githubusercontent.com/2278253/106331441-f314d080-6252-11eb-8aaf-6cef2dff15f7.png">

So technically `12 #foo "hello"` is invalid.  That being the case, I'd say that if we encounter this (a quantity that uses a system-less code), we should log an error.  But for the purposes of `toString()`, we should still represent whatever we have in memory, so `12 #foo "hello"` is the correct stringified representation (even if it is invalid).  Does that make sense?